Example/Source/Configuration.storekit

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Need inputs: Not sure whether we need this `.storekit` configuration file for the example?